charts/telegraf-ds/templates/daemonset.yaml

```
@@ -83,3 +83,5 @@ spec:
       dnsConfig:
 {{ toYaml . | indent 8 }}
       {{- end }}
+      securityContext:
+{{ toYaml .Values.podSecurityContext | indent 8 }}
```

Using `with` prevents rendering empty `podSecurityContext` dictionary.

```suggestion
      {{- with .Values.podSecurityContext }}
      securityContext:
{{ toYaml . | indent 8 }}
      {{- end }}
```